collect_app/src/test/java/org/odk/collect/android/database/DatabaseFilePresenceTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.odk.collect.android.database
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.android.database.forms.DatabaseFormsRepository
+import org.odk.collect.shared.TempFiles.createTempDir
+import java.io.File
+
+@RunWith(AndroidJUnit4::class)
+// https://github.com/getodk/collect/issues/5042
+class DatabaseFilePresenceTest {
```

Comment:
Could this not be a direct test of `DatabaseConnection` so we're not involving a repo?

Reply:
I think you meant: **Could this ~~not~~ be** ?

Reply:
Ah right... I was aware of such constructions in English but I didn't get enough sleep today to catch it probably :) 
Yes it can be DatabaseConnection test.

Reply:
haha yes it definitely does what English does best: be ambiguous. Probably one of those things that makes a lot more sense spoken (with emphasis and tone) than written. I'm also not sure if it's even valid, but native speakers are always the worst at sticking to the rules I guess.